tests/ui/specs/Scroller/ListOfThings/Scroller-GT-28384-specs.js

```diff
@@ -0,0 +1,60 @@
+const {expectFocusedItem} = require('../../VirtualList/VirtualList-utils');
+const ScrollerPage = require('../ScrollerPage');
+
+describe('Scroller List Of Things', function () {
+	beforeEach(function () {
+		ScrollerPage.open('ListOfThings');
+	});
+
+	it('should occur onScroll/onKeydown event [GT-28384], [GT-28385]', function () {
+		// Precondition: Spotlight is on Close button. In this case, 'X' button is replaced 'Top' button.
+		ScrollerPage.buttonTop.moveTo();
+		ScrollerPage.spotlightSelect();
+		expect(ScrollerPage.buttonTop.isFocused()).to.be.true();
+
+		// Step3 -1: 5-way Down several times until reaching to the item which is under the screen.
+		const bottomVisibleIdNum = Number(ScrollerPage.bottomVisibleItemId().slice(4));
+		let index = -1;  // Current focus is on Top button
+		while (index < bottomVisibleIdNum) {
+			ScrollerPage.spotlightDown();
+			ScrollerPage.delay(100);
+			// Step 3-2 Verify: onKeyDown event is added for each 5-way Down.
+			// There is no onKeyDown event from X button to Item 0.
+			expectFocusedItem(++index);
+			expect(Number(ScrollerPage.scroller.getAttribute('data-keydown-events'))).to.be.equal(index);
+		}
+		// Step 3-3,4 Verify: One (1)  onScrollStart/Stop event is added.
+		ScrollerPage.delay(300);
+		expect(Number(ScrollerPage.scroller.getAttribute('data-scrolling-events'))).to.be.equal(1);
+
+		// Step 4-1: 5-way Up several times until reaching to the item which is upper the screen.
+		const topVisibleItemId = Number(ScrollerPage.topVisibleItemId().slice(4));
+		let keydownEvent = 7;
+		while (--index >= topVisibleItemId) {
+			ScrollerPage.spotlightUp();
+			ScrollerPage.delay(100);
+			keydownEvent++;
+
+			// Step 4-2 Verify: onKeyDown event is added for each 5-way Up.
+			expectFocusedItem(index);
+			expect(Number(ScrollerPage.scroller.getAttribute('data-keydown-events'))).to.be.equal(keydownEvent);
+		}
+
+		// Step 4-3,4 Verify: One (1)  onScrollStart/Stop event is added.
+		ScrollerPage.delay(300);
+		expect(Number(ScrollerPage.scroller.getAttribute('data-scrolling-events'))).to.be.equal(2);
+
+		// Perform a test of GT-28385, which is not covered by GT-28384.
+		// 5-way Left a few times.
+		ScrollerPage.spotlightLeft();
+		// One (1) onKeyDown event is added for each 5-way Left.
+		expect(Number(ScrollerPage.scroller.getAttribute('data-keydown-events'))).to.be.equal(keydownEvent + 1);
```

Comment:
```suggestion
		ScrollerPage.spotlightLeft();
		keydownEvent++;
		// One (1) onKeyDown event is added for each 5-way Left.
		expect(Number(ScrollerPage.scroller.getAttribute('data-keydown-events'))).to.be.equal(keydownEvent);
```

Or

```suggestion
		ScrollerPage.spotlightLeft();
		// One (1) onKeyDown event is added for each 5-way Left.
		expect(Number(ScrollerPage.scroller.getAttribute('data-keydown-events'))).to.be.equal(++keydownEvent);
```

To be same with above.

Reply:
Suggested Change.